nextcord/application_command.py

```diff
@@ -2138,6 +2143,14 @@ def __init__(
         self.options: Dict[str, SlashCommandOption] = {}
         self.children: Dict[str, SlashApplicationSubcommand] = {}
 
+    @property
+    def qualified_name(self) -> str:
```

Comment:
```suggestion
    @utils.copy_doc(BaseApplicationCommand.qualified_name)
    def qualified_name(self) -> str:
```
i believe this is needed for overriding a method

Reply:
I think instead of copying the description from the base, this one should have a more detailed description that explains what it will return in case of a parent command etc.

Reply:
```suggestion
    def qualified_name(self) -> str:
        """:class:`str`: Retrieve the command name including all parents space separated.
        
        An example of the output would be `parent group subcommand`.
        """
```
something like this?

Reply:
A `versionadded` can be added to that as well.

Reply:
Yeah